OpenTabletDriver/Driver.cs

```
@@ -197,20 +197,28 @@ protected void InitializeAuxDevice(HidDevice auxDevice, DeviceIdentifier identif
                 Log.Debug("Device", $"Initializing index {index}");
                 auxDevice.GetDeviceString(index);
             }
-            
+
             AuxReader = new DeviceReader<IDeviceReport>(auxDevice, reportParser);
             AuxReader.Report += OnReportRecieved;
 
             if (identifier.FeatureInitReport is byte[] featureInitReport && featureInitReport.Length > 0)
             {
-                Log.Debug("Device", "Setting aux feature: " + BitConverter.ToString(featureInitReport));
-                AuxReader.ReportStream.SetFeature(featureInitReport);
+                try
+                {
+                    AuxReader.ReportStream.SetFeature(featureInitReport);
+                    Log.Debug("Device", $"HID Feature '{BitConverter.ToString(featureInitReport)}' succesfully set");
+                }
+                catch {}
```

Log failed `SetFeature` instead of failing completely silently
```cs
catch
{
    Log.Write("Device", "Failed to set feature: " + BitConverter.ToString(featureInitReport), LogLevel.Warning);
}
```